src/components.d.ts

```diff
@@ -7,6 +7,10 @@
 import { HTMLStencilElement, JSXBase } from "@stencil/core/internal";
 export namespace Components {
     interface PdsIcon {
+        /**
+          * The `alt` attribute of the icon.
+         */
+        "alt"?: string;
```

Comment:
Have we considered using `aria-hidden="true"` instead of alt="" and role="img"? Since we typically use them as just decorative icons, it might avoid sending mixed signals to screen readers.

I briefly tried to find any usage of standalone icons, but haven't so far. Just an additional thought.

Reply:
This can be an option, yes. There's different avenues we can pursue for this. I added a quick meeting on the books to discuss the broader aspect of accessibility in Pine with our inherited attributes utility.